packages/core/content-releases/admin/src/components/ReleasesPanel.tsx

```diff
@@ -51,7 +61,7 @@ const Panel: PanelComponent = ({ model, documentId, collectionType }: PanelCompo
     return null;
   }
 
-  if (releases && releases.length === 0) {
+  if (!releases || (releases && releases.length === 0)) {
```

Comment:
I think this might be the same:
```
if (!releases || releases.length === 0) {
```

If the first condition is met, the second is not computed, so the only way it can get there is by releases not being undefined